PowerUp/app/src/main/res/layout/gamemap.xml

```diff
@@ -39,9 +31,9 @@
         android:id="@+id/BoyfriendButton"
         android:layout_width="@dimen/map_click_width"
         android:layout_height="@dimen/map_click_height"
-        android:layout_above="@id/ContinueButtonMapGameBegins"
         android:layout_alignLeft="@id/HouseButton"
         android:layout_alignStart="@id/HouseButton"
+        android:layout_alignTop="@+id/SchoolButton"
```

Comment:
when i do git commit --amend and then git push origin branch_name, it gives
me that my current branch is behind remote head, requires a pull that leads
to merge conflicts

On Thu, Oct 22, 2015 at 2:02 PM, Alma Castillo Antolin <
notifications@github.com> wrote:

> In PowerUp/app/src/main/res/layout/gamemap.xml
> <https://github.com/systers/powerup-android/pull/4#discussion_r42722953>:
>
> >          android:layout_alignLeft="@id/HouseButton"
> >          android:layout_alignStart="@id/HouseButton"
> > +        android:layout_alignTop="@+id/SchoolButton"
>
> Thanks for highlighting that. You can also switch the order in which you
> declare the Boyfriend and the School button, so you don't need the @+idanymore.
> Since you are using a RelativeLayout, this won't change the behaviour
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/powerup-android/pull/4/files#r42722953>.
>